docs/os-ref.md

```diff
@@ -518,7 +521,8 @@ Forwarders are one of the most important concepts of aragonOS. Rather than hardc
 
 The forwarding interface also allows a frontend interface, like the Aragon client, to calculate "forwarding paths". If you wanted to perform an action but you don't have the required permissions, a client can think of alternative paths for execution. For example, you might be in the Vault app's interface wishing to perform a token transfer. If you only had the permission to create votes, the client would directly prompt you to create a vote rather than let you complete the transfer. The flow is illustrated in the following animation:
 
-![forwarding animation](/docs/assets/fwd.gif)
+<!-- Show animated svg equivalent of assets/fwd.gif -->
+<div id="fwd-call"></div>
```

Comment:
<img width="251" alt="screen shot 2019-01-09 at 12 36 46 am" src="https://user-images.githubusercontent.com/4166642/50884544-687d8200-13a8-11e9-9a0c-0fd9dc739224.png">

Only things I'd change in this image:

- In the DAO ACL Permissions section, using a `:` instead of a `,` to separate the entity
  - It seems like `exec team` is duplicated here?
- For the functions, it make look better to put the closing brace on a new line indented the same as the start, e.g.

```
canPerform(
    param1,
    param2
)
```

Reply:
Right now the section is:

```
DAO ACL Permissions

- exec team, ...
- voting (ANT), ...
- exec team, ...
```

But instead of the `,`, it would probably look nicer as

```
- exec team: ... (merged with the duplicate?)
- voting (ANT): ...
```

Reply:
The Aragon DAC team is out of funds and can't pay for this final revision.  Perhaps the A1 team has funds for this.  If so, you can reach out to @alanborger on keybase.  Here are the source files:

https://docs.google.com/drawings/d/14mGaTC503hjMJNNi60w4KBv91V8huCIB7YseHpjLIy0/edit
https://docs.google.com/drawings/d/1pUg43SrocxqqsaSMwBBe6nVbhBwZcrDiQ9QXrgkgfeI/edit
https://docs.google.com/drawings/d/1PbjL-oK12Lls77SFyjtwbuf9gad8waDQbq8ff8R0IZY/edit
https://docs.google.com/drawings/d/1J_rhbrJksdaOdbPnfgMwWF8nRpwKKj-sfNjAR8urSvc/edit
https://docs.google.com/drawings/d/1eERE8U8XdGS0PyIlzUgamZdPJ1728ffQUOqeb8nL7HA/edit